pr_stats.c

```diff
@@ -2500,11 +2500,11 @@ __print_funct_t stub_print_filesystem_stats(struct activity *a, int curr, int di
 	int i, j;
 	struct stats_filesystem *sfc, *sfm;
 
-
 	if (dis) {
 		printf("\n%-11s  MBfsfree  MBfsused   %%fsused  %%ufsused"
-		       "     Ifree     Iused    %%Iused FILESYSTEM\n",
-		       (dispavg ? _("Summary:") : timestamp[!curr]));
+		       "     Ifree     Iused    %%Iused %s\n",
+		       (dispavg ? _("Summary:") : timestamp[!curr]),
+		       DISPLAY_MOUNT(a->opt_flags) ? _("MOUNTPOINT") : _("FILESYSTEM"));
```

Comment:
MOUNTPOINT and FILESYSTEM should not be enclosed in _() since they are not translated.